src/wrapper.ts

```diff
@@ -24,6 +24,7 @@ if (isEnvVarTrue(LUMIGO_DEBUG)) {
 }
 
 let isTraceInitialized = false;
+let _resource: Resource;
```

Comment:
Actually, I think we do not need to do that! The [Tracer](https://github.com/open-telemetry/opentelemetry-js/blob/db32860bb47d7774a3cf810d5b4805d42a0619ac/packages/opentelemetry-sdk-trace-base/src/Tracer.ts#L40) provides access to the resource!